ios/PurchasesHybridCommon/PurchasesHybridCommon.xcodeproj/project.pbxproj

```diff
@@ -529,7 +529,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = A7ED3A67A7FCB75F4706E21D /* Pods-PurchasesHybridCommon-PurchasesHybridCommonTests.debug.xcconfig */;
 			buildSettings = {
-				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = "$(inherited)";
```

Comment:
fixes a warning coming from cocoapods, who sets values for this automatically, but they'd be overridden by this setting